ios-base/DroidKaigi 2020/About/Views/AboutViewController.swift

```diff
@@ -0,0 +1,165 @@
+import ios_combined
```

Comment:
Is this necessary ? 🤔

Reply:
I have thought there might be a need to use `LangKt`.
In fact, it's unnecessary.
I will remove it.